config/uberon.yml

```diff
@@ -20,28 +20,31 @@ entries:
 
 - exact: /phenoscape-ext.owl
   replacement: http://phenoscape.svn.sourceforge.net/svnroot/phenoscape/trunk/vocab/edit/phenoscape-ext.owl
-  
+
 - exact: /composite-vertebrate.owl
   replacement: https://github.com/obophenotype/uberon/releases/download/v2021-10-01/composite-vertebrate.owl
-  
+
 - exact: /composite-vertebrate.obo
   replacement: https://github.com/obophenotype/uberon/releases/download/v2021-10-01/composite-vertebrate.obo
 
 - exact: /composite-metazoan.owl
   replacement: https://github.com/obophenotype/uberon/releases/download/v2021-10-01/composite-metazoan.owl
-  
+
 - exact: /composite-metazoan.obo
   replacement: https://github.com/obophenotype/uberon/releases/download/v2021-10-01/composite-metazoan.obo
 
 - exact: /experimental/ceph.owl
   replacement: http://purl.obolibrary.org/obo/ceph.owl
 
+- prefix: /subsets/
+  replacement: https://github.com/obophenotype/uberon/releases/latest/download/v
```

Comment:
This one worries me a little, but I trust you know best.

Reply:
iiii the v was wrong, thanks for catching that!